infra/base-images/base-builder/coverage

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -eu
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+cd $OUT
+
+GCS_BUCKET_SUFFIX="-backup.clusterfuzz-external.appspot.com"
+GCS_BACKUP_PATH="/corpus/libFuzzer/"
+
+# This will be used by llvm-cov command to generate the actual report.
+objects=""
+
+gcloud auth login
```

Comment:
let's not do this (logging in and downloading corpus) in the docker image. we can do this in the helper instead,  and mount it into the image. this has the following benefits:

- We don't have to login and re-sync corpus every time
- Builder really shouldn't need a dependency on gcloud
- Decoupling this will make it easier to support this on our infrastructure too.

Reply:
Good point! I've also added multi-threading so we could download multiple corpus backups in parallel rather then iterating 1-by-1.